docs/manual/related-projects.md

```diff
@@ -20,6 +20,11 @@ a tool to snapshot systems on the client side (dpkg/rpm + BTRFS/LVM),
 or a tool to compose on a server and replicate (ChromiumOS, Clear
 Linux).  OSTree is flexible enough to do both.
 
+Note that this section of the docuementation is almost entirely
```

Comment:
Thanks, fixed.  Gotta say it'd be nice if github had a UI that made it easy for the reviewer to push fixups directly to avoid round trips for things like this.